src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
+
+	// The product of the basket
+	static HashMap<String, Integer> map = new HashMap<String, Integer>();
+
+	static String[] localProducts = new String[]{"123", "222", "44", "657"};
+
+	// The fact that the basket has promo code
+	private static boolean codeDePromotion = false;
+
+	public void addProductToBasket(String product, Integer price, boolean isPromoCode) {
```

Comment:
6) The _price_ should not be an integer if it represents euros/dollars as default: something different than X.00 €/$ will never happen (X being an Integer).
I would change it to **BigDecimal** (as long or double can lead to inaccurate numbers when performing calculations).